code/__HELPERS/string_lists.dm

```diff
@@ -12,3 +12,20 @@ GLOBAL_LIST_EMPTY(string_lists)
 		return
 
 	return GLOB.string_lists[string_id] = values
+
+///A wrapper for baseturf string lists, for support for non list values, and a stack_trace if we have major issues
+/datum/proc/baseturfs_string_list(list/values, turf/T)
```

Comment:
```suggestion
/proc/baseturfs_string_list(list/values, turf/baseturf_holder)
```

Reply:
I'll deal with this tomorrow TM

Reply:
actually, better question is why is `string_list` on datum too

Reply:
Ah and I'm uncomfortable with replacing T with baseturf_holder, as it's only used in a crash print, and a var name of that length makes it really hard to read

Reply:
It was implemented that way for some reason.

On the variable name, `T` tells us nothing, besides that it's perhaps some type whose name starts with "t".
Yet this is not a random turf, this is the turf that holds the baseturf string list, for error reporting.
If you dislike the name `baseturf_holder`, as you don't think it conveys appropriate info on what this specific turf is, do please come up with a better name.
Just make it clear. Even if you know what your code does, that does not mean other people who stumble upon it will have that knowledge.

Reply:
That's true, I'm more worried about obscuring the purpose of the print, I'll see what I can do

Reply:
It won't be obscured if the variable has a good name, that's the thing.
If you really want it be known it's a turf, then do something like `baseturf_holder_turf` or something.

Reply:
I'm more worried about line length frankly, I extract like 4 vars for the print, and if the variable name is long, it's gonna get a bit messy